src/pages/feedback-analysis/styles/feedback-analysis.css

```
@@ -235,22 +237,23 @@
 .feedback-analysis__metric-value-row {
   display: flex;
   align-items: baseline;
-  gap: 4px;
-  min-height: 32px;
+  gap: 6px;
+  min-height: 40px;
 }
 
 .feedback-analysis__metric-value {
   font-family: "SUIT", "Pretendard", sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "SUIT" (font-family-name-quotes)

(font-family-name-quotes)